src/build/sign.rs

```diff
@@ -1,30 +1,59 @@
+use armake2::{pbo::PBO, sign::BISignVersion};
 use std::fs;
-use std::fs::DirEntry;
-use std::io::{Error};
+use std::fs::File;
+use std::io::Error;
 use std::path::PathBuf;
 
-use crate::error::*;
+pub fn copy(
+    path_posix: &String,
+    addonsfolder: &String,
+    pbo_filename: &String,
+) -> Result<bool, Error> {
+    fs::copy(path_posix, format!("{}/{}", addonsfolder, pbo_filename))?;
+    Ok(true)
+}
+
+pub fn sign(
```

Comment:
I know this is rustfmt's doing, and the rusty way.

Personally, I am not a fan, but I would be open to switching the entire project to it. That is out of scope here though.

Reply:
Also not a fan.